dangerzone/isolation_provider/base.py

```diff
@@ -49,21 +50,34 @@ def _convert(
     ) -> bool:
         pass
 
-    def print_progress(
+    def _print_progress(
         self, document: Document, error: bool, text: str, percentage: float
     ) -> None:
         s = Style.BRIGHT + Fore.YELLOW + f"[doc {document.id}] "
-        s += Fore.CYAN + f"{percentage}% "
+        s += Fore.CYAN + f"{percentage}% " + Style.RESET_ALL
         if error:
-            s += Style.RESET_ALL + Fore.RED + text
+            s += Fore.RED + text + Style.RESET_ALL
             log.error(s)
         else:
-            s += Style.RESET_ALL + text
+            s += text
             log.info(s)
 
         if self.progress_callback:
             self.progress_callback(error, text, percentage)
 
+    def print_progress_trusted(
+        self, document: Document, error: bool, text: str, percentage: float
+    ) -> None:
+        return self._print_progress(document, error, text, percentage)
+
+    def print_progress(
```

Comment:
If we mark the other as trusted, maybe we should name this untrusted. I feel like untrusted should be the highlighted part.

Reply:
Right, I get that for symmetry, naming it this way makes sense. But I wanted to insert an asymmetry here. Treating progress as untrusted should be the default, hence why I want us to use `print_progress` instead of `print_progress_untrusted` (which is a bit negative as well).

This way, a dev doesn't have to think twice about what function to use. They will use the default one always, and only switch to `print_progress_trusted` if they have a valid reason to do so.